src/mds/mds/input-ui.lisp

```diff
@@ -9,8 +9,10 @@
     (set-input-ui-tk tk)
     (tk-put tk "source ~s" (uw-sfnr "mds/input-ui.tk" :assertIsFile t))
     (let ((user (user-name)))
-      (cond ((equal user "terry")  (tk-put tk "set initialDir mds/investigations/merge-hi-tables/"))
-	    ((equal user "dsmith")  (tk-put tk "set initialDir mds/data/"))
+      (cond ((equal user "terry")     (tk-put tk "set initialDir mds/investigations/merge-hi-tables/"))
+	    ((equal user "dsmith")    (tk-put tk "set initialDir mds/data/"))
+	    ((equal user "stefan")    (tk-put tk "set initialDir ~/Desktop/"))
```

Comment:
It might be better to just have a final catch-all clause like

```lisp
(t (tk-put tk "set initialDir ~/Desktop/"))
```

Do we know that using a `~` in the path actually works?

Reply:
I can confirm that when I go to "Open table or save" after starting lispmds it opens "/Users/stefan/Desktop"